### PR TITLE
normalize pke

### PR DESCRIPTION
**Pull request by matthiasto**
Normalize the pke result: kJ/m²

relKot_c72f73b88f_null_dfa_pke

### CODE REVIEW

File: avaframe/out1Peak/outPlotAllPeak.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ def plotAllPeakFields(avaDir, cfgFLAGS, modName, demData=''):
             # choose colormap
             cmap, col, ticks, norm = pU.makeColorMap(pU.colorMaps[resType], np.amin(data), np.amax(data), continuous=pU.contCmap)
             cmap.set_bad(alpha=0)
-            cmap.set_under(alpha=0)
+            # uncomment this to set the under value for descrete cmap transparent
```
Comment (fso42) [inline avaframe/out1Peak/outPlotAllPeak.py:109]:
```suggestion
            # uncomment this to set the under value for discrete cmap transparent
```

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -2087,15 +2087,15 @@ def exportFields(cfg, Tsave, fieldsList, demOri, outDir, logName):
                 # convert from Pa to kPa
                 resField = resField * 0.001
             if resType == 'pke':
-                # convert from J to kJ
-                resField = resField * 0.001
+                # convert from J/cell to kJ/m²
```
Comment (awirb) [inline avaframe/com1DFA/com1DFA.py:2090]: add to comment - and take actual area of cell into account

File: avaframe/out3Plot/plotUtilsCfg.ini [hunk 3]
```diff
@@ -61,18 +61,18 @@ hillshadeContLevs = 15
...
 unitFM = kg
-unitFTV = $m^2s^{-1}$
-unitVx = $ms^{-1}$
-unitVy = $ms^{-1}$
-unitVz = $ms^{-1}$
+unitFTV = $m^2.s^{-1}$
+unitVx = $m.s^{-1}$
+unitVy = $m.s^{-1}$
+unitVz = $m.s^{-1}$
 unitTA = °
```
Comment (awirb) [inline avaframe/out3Plot/plotUtilsCfg.ini:76]: why these dots? I think if something than an space would be nice..

**Review by awirb — CHANGES_REQUESTED** [review]

**Review by fso42 — APPROVED** [review]

File: avaframe/com1DFA/com1DFA.py [hunk 4]
```diff
@@ -2047,7 +2047,7 @@ def readFields(inDir, resType, simName='', flagAvaDir=True, comModule='com1DFA',
     return fieldsList, fieldHeader, timeList
 
 
-def exportFields(cfg, Tsave, fieldsList, demOri, outDir, logName):
+def exportFields(cfg, Tsave, fieldsList, dem, outDir, logName):
```
[the remark this answers is not in the document]
Comment (matthiasto) [reply avaframe/com1DFA/com1DFA.py:2050]: I need the real area of the dem cells, this is only available in dem no?

**Pull request merged**